src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -34,6 +35,7 @@ export const AddressInput = ({ rules, placeholder, enableQr = false }: AddressIn
     }) => (
       <Input
         spellCheck={false}
+        autoFocus={!isMobile}
```

Comment:
@reallybeard `const isMobile = Boolean(window?.isShapeShiftMobile)` and we probably want to disable this for all mobile users, not just mobile app - happy to keep the previous diff and remove it algotether not to break mobile